index.html

```diff
@@ -19,42 +20,47 @@
 <header id="top">
     <nav>
         <ul>
-            <p><a herf=index.html>Home</a></p>
-            <p><a herf=links.html>Links</a></p>
-            <p><a herf=glossary.html>glossary</a></p>
-            </ul>
-            </nav> </header>
-            <main> </main>
-            <footer> </footer>
-            ◤<a href="Index.html"><title>Back To Top</title></a>       
-        </div>
+            <li><a href=index.html>Home</a></li>
+            <li><a href=links.html>Links</a></li>
+            <li><a href=glossary.html>Glossary</a></li>
+        </ul>
+    </nav>
+</header>
+<main>
     <h1>Welcome to Matthew Burgan's CSD120 Website </h1>
-  <p> Hi my name is Matthew I decided to take this program at sault college cause I really enjoy working with Computers and I enjoyed the coding I did in highscool alot and im excited to see what I can learn and accomplish from this course.
-    Some of my favorite things to do when im not busy with schoolwork is hangout with friends and go walking with them. I also Enjoy magic the gathering the card game and just the usual video games.</p>
+    <p>Hi my name is Matthew I decided to take this program at sault college cause I really enjoy working with Computers and I enjoyed the coding I did in highscool alot and im excited to see what I can learn and accomplish from this course.</p>
+    <p>Some of my favorite things to do when im not busy with schoolwork is hangout with friends and go walking with them. I also Enjoy magic the gathering the card game and just the usual video games.</p>
     <h2>My Favorite things </h2>
     <a href="https://g2esports.com/">
-    <img src="./images/logo.png" alt=G2 esports team logo Style="Width:42px;height :42px;">
-</a>
+        <img src="./images/logo.png" alt="G2 esports team logo" style="width:42px;height :42px;">
+    </a>
```

Comment:
It's best practice to always put attribute values inside quotation marks for exactly the reason that you see here: attribute values with spaces in them can cause problems when not quoted.